homeassistant/components/binary_sensor/wink.py

```diff
@@ -0,0 +1,78 @@
+"""
+Support for Wink sensors.
+
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.wink/
+"""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import CONF_ACCESS_TOKEN, STATE_ON, STATE_OFF
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['python-wink==0.6.1']
+
+BINARY_TYPES = ['brightness',
```

Comment:
See the bloomsky binary_sensor for an example.

All possible classes are in binary_sensor __init__

On Tue, Feb 23, 2016 at 2:07 PM, w1ll1am23 <notifications@github.com> wrote:

> In homeassistant/components/binary_sensor/wink.py
> <https://github.com/balloob/home-assistant/pull/1380#discussion_r53858818>
> :
>
> > @@ -0,0 +1,78 @@
> > +"""
> > +Support for Wink sensors.
> > +
> > +For more details about this platform, please refer to the documentation at
> > +at https://home-assistant.io/components/sensor.wink/
> > +"""
> > +import logging
> > +
> > +from homeassistant.components.binary_sensor import BinarySensorDevice
> > +from homeassistant.const import CONF_ACCESS_TOKEN, STATE_ON, STATE_OFF
> > +from homeassistant.helpers.entity import Entity
> > +
> > +REQUIREMENTS = ['python-wink==0.6.1']
> > +
> > +BINARY_TYPES = ['brightness',
>
> Can you provide an example of this? And what icons should I get? This is
> what they look like now.
> [image: screenshot - 02232016 - 05 06 16 pm]
> <https://cloud.githubusercontent.com/assets/6432770/13268310/d6777904-da4f-11e5-8792-cc8538580987.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/1380/files#r53858818>.
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.